### PR TITLE
Better error message precision

### PR DESCRIPTION
**Pull request by gre**
- Receive: differenciate between wrong app & wrong account
- unify isCurrencyAppOpened back into getAddress (making it more strong for bitcoin related apps)

### CODE REVIEW

File: src/components/DeviceConnect/index.js [hunk 1]
```diff
@@ -311,14 +309,7 @@ class DeviceConnect extends PureComponent<Props> {
...
             <Box shrink selectable>
-              {accountName ? (
-                <Trans i18nKey="deviceConnect:info" parent="div">
-                  {'You must use the device associated to the account '}
-                  <strong>{accountName}</strong>
-                </Trans>
-              ) : (
-                String(errorMessage || '')
-              )}
+              {String(errorMessage || '')}
```
Comment (meriadec) [inline src/components/DeviceConnect/index.js]: so, no more markup inside error message? isn't it a regression?

File: src/components/EnsureDeviceApp/index.js [hunk 2]
```diff
@@ -116,31 +117,42 @@ class EnsureDeviceApp extends PureComponent<Props, State> {
...
+          .catch(e => {
+            if (
+              e &&
+              (e.name === 'TransportStatusError' ||
+                // we don't want these error to appear (caused by usb disconnect..)
+                e.message === 'could not read from HID device' ||
+                e.message === 'Cannot write to HID device')
+            ) {
+              logger.log(e)
+              throw new Error(`You must open application ‘${cur.name}’ on the device`)
```
Comment (meriadec) [inline src/components/EnsureDeviceApp/index.js]: is it intended to use french quotes here?

Comment (gre) [reply src/components/EnsureDeviceApp/index.js]: not really. what do you think we should use?

Comment (gre) [reply src/components/EnsureDeviceApp/index.js]: ““ maybe ?

Comment (gre) [reply src/components/EnsureDeviceApp/index.js]: ''
""
‘‘
““

Comment (meriadec) [reply src/components/EnsureDeviceApp/index.js]: After all, let's keep it like that. It's pretty elegant :nail_care:

File: src/components/DeviceConnect/index.js (hunk 1 above)
Comment (gre) [reply src/components/DeviceConnect/index.js]: we just let the error flow whatever it is, which allow more error cases to appear. we'll just try to skip errors we don't want but ultimately we have more precise error

Comment (gre) [reply src/components/DeviceConnect/index.js]: BTW in future, we'll make this displaying on only one line and with ellipsis `...`. the error also need to be selectable

**Pull request merged**